<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poste</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBEnvoyer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBValider">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInformation">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>La Poste</string>
    </property>
   </widget>
   <widget class="QLabel" name="lAffranchissement">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Affranchissement : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoNombre">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de Colis :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoPoids">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poids du Colis :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoPrix">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix du Colis :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoPrixTotal">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prix total du/des Colis :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lPoids">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lInfoType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type de Colis :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lPrix">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lPrixTotal">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lTypeColis">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>145</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Veuillez saisir le type de colis : </string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>125</width>
      <height>72</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="rBSuivis">
     <property name="text">
      <string>Lettre Suivis</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBVerte">
     <property name="text">
      <string>Lettre Verte</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBRecommander">
     <property name="text">
      <string>Lettre Recommand√©e</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBPrioritaire">
     <property name="text">
      <string>Lettre Prioritaire</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lPoids_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>gramme</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPoidsColis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>238</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Veuillez saisir le poids de votre colis (en gramme) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEPoids">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lRecapitulatif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>218</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recapitulatif du colis n' :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNumColis">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lENombreColis">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lNbColis">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>156</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Veuillez saisir le nombre de colis :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
